saleor/plugins/vatlayer/plugin.py

```diff
@@ -118,10 +174,20 @@ def _get_taxes_for_country(self, country: Country):
         from cache or db.
         """
         if not country:
-            country = Country(settings.DEFAULT_COUNTRY)
+            source_country_code = self.config.source_country or settings.DEFAULT_COUNTRY
```

Comment:
There is also the `company_address` defined in the SiteSettings model, maybe we want to use it here as well?